$NetBSD$

Broken detection of header location of netpbm.
Kill it and pass with CONFIGURE_ENV instead.

--- configure.orig	2008-10-14 22:39:03.000000000 +0000
+++ configure
@@ -19908,16 +19908,6 @@ fi
 
 fi
 
-if test "$ppm_support" = "yes"; then
-
-   NETPBM_PPMH=`find /usr/include /usr/local/include/ $CPATH -name ppm.h`
-   if test -n "$NETPBM_PPMH"; then
-        NETPBM_CFLAGS="-I ${NETPBM_PPMH%%ppm.h}"
-   else
-        ppm_support="no"
-   fi
-fi
-
 else
   ppm_support="no"
 fi
